pkg/backup/backup/backup_manager.go

```diff
@@ -318,8 +318,8 @@ func (bm *backupManager) makeBackupJob(backup *v1alpha1.Backup) (*batchv1.Job, s
 	if tc.Spec.TiDB.TLSClient != nil && tc.Spec.TiDB.TLSClient.Enabled && !tc.SkipTLSWhenConnectTiDB() {
 		args = append(args, "--client-tls=true")
 		clientSecretName := util.TiDBClientTLSSecretName(backup.Spec.BR.Cluster)
-		if backup.Spec.From.TLSClient != nil && backup.Spec.From.TLSClient.TLSSecret != "" {
-			clientSecretName = backup.Spec.From.TLSClient.TLSSecret
+		if backup.Spec.From.TLSClientSecretName != nil {
```

Comment:
Also check the length of *backup.Spec.From.TLSClientSecretName?

Reply:
The same comment apply for the code in the other files.

Reply:
No need to check it here.